packages/react-ui/src/theme/sections/components/buttons.js

```diff
@@ -1,15 +1,13 @@
+import { createTheme } from '@mui/material/styles';
 import { getSpacing } from '../../themeUtils';
-import { commonPalette } from '../palette';
-import { themeTypography } from '../typography';
-import { themeShadows } from '../shadows';
 import { ICON_SIZE_MEDIUM, ICON_SIZE_LARGE } from '../../themeConstants';
 
 const sizeSmall = getSpacing(3);
 const sizeMedium = getSpacing(4);
 const sizeLarge = getSpacing(6);
 const radius = getSpacing(0.5);
 
-export const buttonsOverrides = {
+export const buttonsOverrides = createTheme({
```

Comment:
createTheme ? I wouldn't expect this here

Reply:
Yes, it was just a test I did base on some docs I read, but I'm going to revert it